File: M3Tools/Controls/SummaryCtrl.vb
Public Class SummaryCtrl
	<ComponentModel.Browsable(False)>
	<ComponentModel.Category("Data")>
	Property Display As Object
		Get
			Return pg_Summary.SelectedObject
		End Get
		Set(value As Object)
			pg_Summary.SelectedObject = value

			If Not IsNothing(value) Then
				pg_Summary.ExpandAllGridItems()
			End If
		End Set
	End Property

	' TODO: Create custom summary page for this. No controls provide desired look and feel

	'' Summary
	''	* Basics
	''		- Name:	Customer Name
	''		- Email: Customer Email
	'' * Address
	''		- Street: Customer Street Address
	''		- City: Customer City
	''		- State: Customer State
	''		- Zip Code: Customer Zip Code
End Class

File: M3Tools/Controls/SummaryCtrl.cs

namespace SPPBC.M3Tools
{
	/// <summary>
	/// 
	/// </summary>
    public partial class SummaryCtrl
    {
		/// <summary>
		/// The data to display in the summary
		/// </summary>
        [System.ComponentModel.Browsable(false)]
        [System.ComponentModel.Category("Data")]
        public object Display
        {
            get
            {
                return pg_Summary.SelectedObject;
            }
            set
            {
                pg_Summary.SelectedObject = value;

                if (!(value == null))
                {
                    pg_Summary.ExpandAllGridItems();
                }
            }
        }

		/// <summary>
		/// 
		/// </summary>
        public SummaryCtrl()
        {
            InitializeComponent();
        }

        // TODO: Create custom summary page for this. No controls provide desired look and feel

        // ' Summary
        // '	* Basics
        // '		- Name:	Customer Name
        // '		- Email: Customer Email
        // ' * Address
        // '		- Street: Customer Street Address
        // '		- City: Customer City
        // '		- State: Customer State
        // '		- Zip Code: Customer Zip Code
    }
}
